core/src/com/group66/game/logging/FileLogger.java

```diff
@@ -0,0 +1,35 @@
+package com.group66.game.logging;
+
+import java.util.Date;
+
+import org.apache.commons.lang.time.DateFormatUtils;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.files.FileHandle;
+
+public class FileLogger extends Logger {
+    private static FileHandle handle;
+
+    /**
+     * Constructor for a file logger
+     * @param mt the message type to write
+     */
+    public FileLogger(MessageType mt){
+        String date = DateFormatUtils.format(new Date(), "yyyyMMdd-HHmmss");
```

Comment:
Change format to `yyyyMMddHHmmss` please, as that was what I put in the requirements.

I also put into the requirements: 
> The logger must provide timestamps in front of all the messages

and

> The logger should log in the line format: `[timestamp] LEVEL: message`

So could you move the timestamp to the abstract class, make the message as described in the requirements:
`[timestamp] LEVEL: message`
And just pass that to `write()`